### DISCUSSION TITLE
"QueuePool limit" error on webserver pod with helm chart on GCP

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17513] [Q&A]
Hey everyone, I'm using the official helm chart on GCP. Getting this error from the webserver pod very frequently in conjunction with timing out graphql requests:
```Retrying failed database connection: QueuePool limit of size 1 overflow 10 reached```
any ideas?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C014N0PK37E/p1698747107037659?thread_ts=1698747107.037659&cid=C014N0PK37E).*

**Comment by NiallRees** [comment]
This was asked by me - any help greatly appreciated.

**Comment by sryza** [comment]
This looks related to https://github.com/dagster-io/dagster/issues/17427
